website_rma/views/request_rma_form.xml

```diff
@@ -124,9 +124,11 @@
                                                 class="form-control o_website_form_input"
                                                 name="description"
                                                 required=""
-                                            ><t
+                                            >
+                                                <t
                                                     t-esc="request.params.get('description', '')"
-                                                /></textarea>
+                                                />
+                                            </textarea>
```

Comment:
Beware of reformatting `<textarea>` tags, as they behave like `<pre>` so you've got this empty space from the xml declaration:

![image](https://user-images.githubusercontent.com/5040182/188867663-00e33506-c3e6-49c8-81bc-d231316b7786.png)